### ISSUE TITLE
changing the dimensionality of vector in `Lexeme`

### ISSUE THREAD

**Issue by michaelcapizzi** [explosion/spaCy#456]
I read the tutorial [here](https://spacy.io/docs/tutorials/load-new-word-vectors) about updating the `vector` for a given `Lexeme`.

But it appears that the resulting `vector` must *still be* of size 300:

```
>>>en_nlp.vocab["apple"].vector = np.array([1,2,3])
AssertionError                            Traceback (most recent call last)
<ipython-input-295-5194f6bac79a> in <module>()
----> 1 en_nlp.vocab["apple"].vector = np.array([1,2,3])

.../lib/python3.5/site-packages/spacy-0.101.0-py3.5-linux-x86_64.egg/spacy/lexeme.pyx in spacy.lexeme.Lexeme.vector.__set__ (spacy/lexeme.cpp:4702)()

AssertionError: 
```

I'm trying to reduce the dimensionality of word vectors for use in a particular machine learning process.

Is there any way to update the size of the `vector`s in `spaCy` to be of size **other than** 300?

I tried this too:

```
en_nlp.vocab.vectors_length = 3
AttributeError                            Traceback (most recent call last)
<ipython-input-298-275cbd8a1fed> in <module>()
----> 1 en_nlp.vocab.vectors_length = 3

AttributeError: attribute 'vectors_length' of 'spacy.vocab.Vocab' objects is not writable
```

**Comment by bobcolner** [comment]
I've used PCA to reduce the dimensionality of the word vectors: 

See:
http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.IncrementalPCA.html#sklearn.decomposition.IncrementalPCA

**Comment by honnibal** [comment]
This should be fixed in 1.1.0 — please reopen if it's not!
